### PR TITLE
Release template

### PR DESCRIPTION
**Pull request by ScharfViktor**
- I created release template like in the client https://github.com/owncloud/client/blob/master/.github/release_template.md
Separately, I put the out existing smoke tests to https://github.com/owncloud/QA and link it
Note: some links will be available after merging https://github.com/owncloud/QA/pull/813

### CODE REVIEW

File: .github/release_template.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+* [ ] Brief company-wide heads up via mail @tbsbdr
+* [ ] DEV: Create branch `release-x.x.x`
+  * [ ] DEV: bump ocis version in necessary files
+  * [ ] DEV: `ocis-pkg/version/version.go`
+  * [ ] DEV: `sonar-project.properties`
+  * [ ] DEV: released deployment versions
+  * [ ] DEV: prepare changelog folder in `changelog/x.x.x_???`
+* [ ] Release Notes @tbsbdr
+* [ ] Migration Doc @mmattel
```
Comment (mmattel) [inline .github/release_template.md:63]:
```suggestion
* [ ] Release Notes + Breaking Changes @tbsbdr
* [ ] Migration + Breaking Changes Admin Doc @mmattel
* [ ] Migration + Breaking Changes Helm Chart Doc @wkloucek @mmattel
```

File: .github/release_template.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  * [ ] QA: [ldap](ocis.ocis-ldap.released.owncloud.works)
+* [ ] QA: e2e with different storage:
+  * [ ] QA: local
+  * [ ] QA: nfs
+  * [ ] QA: s3
+* [ ] QA: Different clients:
+  * [ ] QA: desktop (define version) https://github.com/owncloud/client/releases
+  * [ ] QA: against mac - smoke test
+  * [ ] QA: against windows - smoke test
+  * [ ] QA: against linux (use auto tests)
```
Comment (individual-it) [inline .github/release_template.md:45]:
```suggestion
  * [ ] QA: desktop (define version) https://github.com/owncloud/client/releases
      * [ ] QA: against mac - smoke test
      * [ ] QA: against windows - smoke test
      * [ ] QA: against linux (use auto tests)
```

**Review by individual-it — CHANGES_REQUESTED** [review]
just small formatting to make it clear that the mac/win/linux belongs to "desktop"
The rest looks good :+1:

File: .github/release_template.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+* [ ] DEV: bump released deployments to `v3.0.0-rc.1`
+* [ ] DEV: update `ocis.released.owncloud.works`
+* [ ] DEV: https://ocis.team.owncloud.works/
+  * [ ] DEV: needs snapshot and migration
+
+### QA Phase
+
+* [ ] QA: Confirmatory testing (if needed)
+* [ ] QA: [Compatibility test](https://github.com/owncloud/QA/blob/master/Server/Test_Plan_OCIS_Compatibility_Test.md)
+* [ ] QA: [Perfomance test](https://github.com/owncloud/cdperf/tree/main/packages/k6-tests/src)
```
Comment (phil-davis) [inline .github/release_template.md:27]:
```suggestion
* [ ] QA: [Performance test](https://github.com/owncloud/cdperf/tree/main/packages/k6-tests/src)
```

File: .github/release_template.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+* [ ] Merge release notes 
+
+### Post-release communication
+* [ ] DEV: Create a docs-stable-3.0 branch based on the docs folder
+* [ ] DEV/QA: Ping documentation in RC about the new release tag (for ocis version bump in docs)
+* [ ] DEV/QA: Ping marketing to update all download links (download mirrors are updated at the full hour, wait with ping until download is actually available)
+* [ ] DEV/QA: Ping @hodyroff once the demo instances are running this release
+* [ ] DEV/QA: notify @michaelstingl @hosy @fmoc @jesmrec to publish client finals
+* [ ] DEV: Merge back release branch
+* [ ] DEV: Create stable-3.0 branch from final tag
```
Comment (mmattel) [inline .github/release_template.md:76]:
```suggestion
### Post-Release Communication
* [ ] DEV: Create a docs-stable-x.y branch based on the docs folder in the ocis repo @micbar 
* [ ] DEV: Create a x.y.z release in the ocis-helm repo (frozen state) @wkloucek
* [ ] DEV/QA: Ping documentation in RC about the new release tag (for ocis/helm chart version bump in docs)
* [ ] DEV/QA: Ping marketing to update all download links (download mirrors are updated at the full hour, wait with ping until download is actually available)
* [ ] DEV/QA: Ping @hodyroff once the demo instances are running this release
* [ ] DEV/QA: notify @michaelstingl @hosy @fmoc @jesmrec to publish client finals
* [ ] DEV: Merge back release branch
* [ ] DEV: Create stable-x.y branch in the ocis repo from final tag
```
Some small improvements and clarifications.

**Review by phil-davis — APPROVED** [review]

File: .github/release_template.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  * [ ] DEV: bump ocis version in necessary files
+  * [ ] DEV: `changelog/CHANGELOG.tmpl`
+  * [ ] DEV: `ocis-pkg/version/version.go`
+  * [ ] DEV: `sonar-project.properties` 
+  * [ ] DEV: prepare changelog folder in `changelog/x.x.x_????_??_??`
+* [ ] DEV: Check successful CI run on release branch
+* [ ] DEV: Create signed tag `v3.0.0-rc.1`
+* [ ] DEV: Check successful CI run on `v3.0.0-rc.1` tag / BLOCKING for all further activity
+* [ ] DEV: Merge back release branch
+* [ ] DEV: bump released deployments to `v3.0.0-rc.1`
```
Comment (mmattel) [inline .github/release_template.md:18]:
```suggestion
* [ ] DEV: Create signed tag `vx.y.z-rc.1`
* [ ] DEV: Check successful CI run on `vx.y.z-rc.1` tag / BLOCKING for all further activity
* [ ] DEV: Merge back release branch
* [ ] DEV: bump released deployments to `vx.y.z-rc.1`
```

**Review by mmattel — APPROVED** [review]

**Review by individual-it — APPROVED** [review]

**Pull request merged**